templates/cluster/eks/Chart.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
+name: eks
+description: |
+  An HMC template to deploy a cluster on EKS with bootstrapped control plane nodes.
+type: application
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.0.1
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "1.30.4"
```

Comment:
probably shouldn't be in the appVersion if we configure k8s version in the values. so let it be just 0.0.1

Reply:
On other templates we put k0s version in the `appVersion` for visibility. If this isn't required anymore we should make in uniform and probably remove the field completely

Reply:
it makes sense for other providers as it represents the k0s version (and we decided to not make k8s versions configurable through values and keep k8s upgrades between templates)

Reply:
Ok, got it. Let's remove `appVersion` from this chart then. IMO it's better than fake `0.0.1` version.